src/dbcp/extract/gridstatus_isoqueues.py

```diff
@@ -20,21 +20,23 @@
 # except for spp and ISONE because the recent versions
 # have columns the old versions don't.
 ISO_QUEUE_VERSIONS: dict[str, str] = {
-    "miso": "1704654953145483",
-    "caiso": "1704654953474846",
-    "pjm": "1704654953842777",
-    "ercot": "1704654954177109",
-    "spp": "1704654954488739",
-    "nyiso": "1702235705611699",
-    "isone": "1704654954804863",
+    "miso": "1709776341526904",
+    "caiso": "1709776259365612",
+    "pjm": "1709776387308475",
+    "ercot": "1709776286037540",
+    "spp": "1709776417253050",
+    "nyiso": "1709776369660949",
+    "isone": "1709776310995150",
```

Comment:
These are the same files just new revision numbers. Copying the parquet GS archives to the bucket generated new revision numbers. I linked the old and new revision numbers using the file hash generated by GCS.